data/campaigns/Descent_Into_Darkness/scenarios/12_Endless_Night.cfg

```diff
@@ -458,6 +565,197 @@
+                                message= _ "But it seems to be hurting the lich. Grab it, retreat and throw it on a bigger fire. Once we've burnt it, maybe the lich will be weaker."
+                            [/message]
+                        [/case]
+
+                        [case]
+                            value=elf
+                            [message]
+                                speaker=unit
+                                message= _ "That's an evil grimoire!"
+                            [/message]
+
+                            [message]
+                                speaker=Foolish Hero
+                                message= _ "Bring it with us, and let's retreat. Once it's burned with faerie fire, maybe the lich will be weakened."
+                            [/message]
+                        [/case]
+
+                        [case]
+                            value=dwarf
+                            [message]
+                                speaker=unit
+                                message= _ "That's an evil grimoire, bound in iron."
+                            [/message]
+
+                            [message]
+                                speaker=Foolish Hero
+                                message= _ "Destroy it."
+                            [/message]
+
+                            [message]
+                                speaker=Mal Keshar
+                                message= _ "<small>(faking pain)</small> Aaargh!"
+                            [/message]
+
+                            [message]
+                                speaker=unit
+                                message= _ "My hammer can't dent it and my axe can't cut the paper."
+                            [/message]
+
+                            [message]
+                                speaker=Foolish Hero
+                                message= _ "But it seems to be hurting the lich. Bring it with us, retreat and let the forge destroy it."
+                            [/message]
+                        [/case]
+
+                        [case]
+                            value=orc
+                            [message]
+                                speaker=unit
+                                message= _ "The skull on that book looks good."
+                            [/message]
+
+                            [message]
+                                speaker=Foolish Hero
+                                message= _ "Idiot."
+                            [/message]
+
+                            [message]
+                                speaker=Mal Keshar
+                                message= _ "<small>(faking pain)</small> Aaargh!"
+                            [/message]
+
+                            [message]
+                                speaker=unit
+                                message= _ "Uuuuuh, I can't pull the skull off the book, but lich screams when I try."
```

Comment:
There's a word missing before "lich". I'm planning to push a version with this typo corrected and a few more translation hints added, that will be either today or tomorrow.